boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -49,11 +49,22 @@
 			   <16 0 &gpio1 12 0>,	/* D10 */
 			   <17 0 &gpio0 9 0>,	/* D11 */
 			   <18 0 &gpio0 10 0>,	/* D12 */
-			   <19 0 &gpio0 8 0>,	/* D13 */
+			   <19 0 &gpio0 28 0>,	/* D13 */
```

Comment:
This should be left the way it was, which is correct according to the Audio DK (PCA10121) Schematics. Why is this moved?

Reply:
My bad, I looked at the silkscreen, and apparently my eyes are not what they sued to be :-)